package.json

```diff
@@ -6,7 +6,8 @@
     "start": "ng serve",
     "prebuild": "npm run update-useragent",
     "build": "ng build",
-    "build:prod": "ng build --prod",
+    "build:prod": "ng build --prod && npm run build:schematics",
```

Comment:
Are you sure the order is correct and we should build the schematics **after** we build the code for production?

Also, do we want to include any of these prod build tasks on the CI? e.g. to ensure it transpiles well

Reply:
Yes the order is correct. The schematics build process just compiles the TypeScript and then copies a bunch of files into the build directory. So the build directory has to exist for the schematics step to work, hence the build order.

We could add it to CI; I was thinking we could add it to the `npm build` step too which would cover that. What do you think?

Reply:
Yes, that makes sense @stevehobbsdev . 👍